documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -339,49 +329,87 @@ Model In Image model files, application archives, and the WDT installation files
    In a later step, we will specify a domain resource `domain.spec.configuration.model.modelHome`
    attribute that references this directory.
 
-1. Run `docker build` using `/tmp/mii-sample/ai-docker-file/Dockerfile` to create your auxiliary
-   image using a small `busybox` image as the base image.
-
-   ```shell
-   $ docker build -f /tmp/mii-sample/ai-docker-file/Dockerfile \
-     --build-arg AUXILIARY_IMAGE_PATH=/auxiliary \
-     --tag model-in-image:WLS-AI-v1 .
-   ```
-
-   See `./Dockerfile` for an explanation of each build argument.
-
-   {{%expand "Click here to view the Dockerfile." %}}
-   ```
-   # Copyright (c) 2021, Oracle and/or its affiliates.
-   # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-
-   # This is a sample Dockerfile for supplying Model in Image model files
-   # and a WDT installation in a small separate auxiliary
-   # image. This is an alternative to supplying the files directly
-   # in the domain resource `domain.spec.image` image.
-
-   # AUXILIARY_IMAGE_PATH arg:
-   #   Parent location for Model in Image model and WDT installation files.
-   #   Must match domain resource 'domain.spec.auxiliaryImageVolumes.mountPath'
-   #   For model-in-image, the following two domain resource attributes can
-   #   be a directory in the mount path:
-   #     1) 'domain.spec.configuration.model.modelHome'
-   #     2) 'domain.spec.configuration.model.wdtInstallHome'
-   #   Default '/auxiliary'.
-   #
-
-   FROM busybox
-   ARG AUXILIARY_IMAGE_PATH=/auxiliary
-   ARG USER=oracle
-   ARG USERID=1000
-   ARG GROUP=root
-   ENV AUXILIARY_IMAGE_PATH=${AUXILIARY_IMAGE_PATH}
-   RUN adduser -D -u ${USERID} -G $GROUP $USER
-   COPY ./ ${AUXILIARY_IMAGE_PATH}/
-   RUN chown -R $USER:$GROUP ${AUXILIARY_IMAGE_PATH}
-   USER $USER
-   ```
-   {{% /expand %}}
+1. Use one of the two options listed below to create the auxiliary image using a small `busybox` image as the base image. 
+   - Option 1 - Use the `createAuxImage` option of the [_WebLogic Image Tool_ (WIT)](https://oracle.github.io/weblogic-image-tool/) to create the auxiliary image.
+     See [Create Auxiliary Image](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-aux-image/) for more details.
+
+     You've already set up the WebLogic Image Tool in the `/tmp/mii-sample/model-images` directory and added the WDT installer to the ImageTool cache during the prerequisite steps. 
```

Comment:
Remove "You've already...steps.", this is implied by the "If you don't see the..." sentence which already follows.